src/sentry/static/sentry/app/components/modals/sentryAppPublishRequestModal.tsx

```diff
@@ -143,8 +143,13 @@ export default class SentryAppPublishRequestModal extends React.Component<Props>
     this.props.closeModal();
   };
 
-  handleSubmitError = () => {
-    addErrorMessage(t('Request to publish %s fails.', this.props.app.slug));
+  handleSubmitError = err => {
```

Comment:
maybe you could add a type? :) 

`type OnSubmitError = Parameters<NonNullable<Form['props']['onSubmitError']>>;
`
```suggestion
  handleSubmitError = (error: OnSubmitError[0],  _instance: OnSubmitError[1], _id: OnSubmitError[2] ) => {
```

Reply:
@priscilawebdev But the error value has an `any` type on Form `onSubmitError` error arg. And all we want is just the error value. Do we need to type unused variables?

Reply:
I understand, but I would still add the types. Maybe in the future, we can replace the type any of the forms with a better type and it will be automatically replicated here. But it's just a suggestion 😉  Frontend looks good.